### ISSUE TITLE
Experiment: Next.js via custom server.

### ISSUE THREAD

**Issue by ryan-roemer** [FormidableLabs/nextjs-serverless-demo#3]
- [x] Follow the [custom server](https://nextjs.org/docs/advanced-features/custom-server) Next.js guide with `serverless-http`
- [x] Review packaging with Jetpack to make sure we don't bring in all the dev stuff we don't need.
- [x] Verify it actually works.

**Comment by ryan-roemer** [comment]
So far in branch https://github.com/FormidableLabs/nextjs-serverless-demo/tree/experiment/nextjs-custom-server there is a lot of cold start latency in the Next.js app. It needs `prepare()` which adds 3s on my machine and lazy compiles each page at varying latencies:

```
$ yarn start
yarn run v1.22.5
$ eval $(yarn -s env) && node server/index.js
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
event - compiled successfully
{ message: 'Next.js prepare() time', elapsedMs: 3398 }
Server started at http://0.0.0.0:3000
event - build page: /
wait  - compiling...
event - compiled successfully
event - build page: /posts/[id]
wait  - compiling...
event - compiled successfully
```

**Comment by ryan-roemer** [comment]
Implemented in #9
